.remarkrc.js

```diff
@@ -1,62 +1,56 @@
-module.exports = {
-  "settings": {
-    "bullet": "*",
-    "emphasis": "_",
-    "strong": "*",
-    "fence": "`",
-    "rule": "-",
-    "ruleSpaces": false,
-    "ruleRepetition": 3,
-    "gfm": true,
-    "footnotes": true,
-    "commonmark": false,
-    "pedantic": false,
-    "entities": false,
-    "fences": true,
-    "spacedTable": true,
-    "listItemIndent": "💩",
-    "remarkLinstListItemIndent": "💩",
+exports.plugins = {
+  settings: {
+    bullet: "*",
+    emphasis: "_",
+    strong: "*",
+    fence: "`",
+    rule: "-",
+    ruleSpaces: false,
+    ruleRepetition: 3,
+    gfm: true,
+    footnotes: true,
+    commonmark: false,
+    pedantic: false,
+    entities: false,
+    fences: true,
+    spacedTable: true,
+    listItemIndent: "💩",
+    remarkLinstListItemIndent: "💩",
```

Comment:
💩❓

Reply:
It's what all these libraries use for settings that are not valid. Tese settings should give us errors when codacy tries to use them. But they do not.